### ISSUE TITLE
A build with --enable-etsf-io fails if the Yambo has to download etsf-io itself in Yambo 4.5.3

### ISSUE THREAD

**Issue by klust** [yambo-code/yambo#29]
The build process fails to download the etsf-io package. TThe cause is a download failure, leaving an empty file.

The download is done by the Makefile in ``lib/archive``. On our system, this executes
the command
```bash
wget --no-check-certificate -O etsf_io-1.0.4.tar.gz https://github.com/yambo-code/yambo/files/845218/
```
This is not what is intended, and the cause is a typo in package.list. Line 56 (line number for Yambo 4.5.3) should
read
```bash
url_etsf_io=https://github.com/yambo-code/yambo/files/845218/$(tarball_etsf_io)
```
(the ``_io`` at the end is missing).

The mistake in package.list is still present in the version that I found in the repo on 3 February 2020, but is now on line 70.

**Comment by sangallidavide** [comment]
Thanks for reporting this.

The reason why it was left there is that etsf-io library is in practice not used anymore and should be removed (together with the --enable-etsf-io option) in the future

I'll fix the issue anyway to avoid confusion.

**Comment by sangallidavide** [comment]
Fixed in branch 4.5 with commit 4ae3985
